### ISSUE TITLE
Presets system does not work with Daily Builds of blender

### ISSUE THREAD

**Issue by HarmZero** [PhysicalAddons/physical-starlight-and-atmosphere#38]
**Describe the bug**
[Presets system does not work with Daily Builds of blender, I have to work with Daily Builds So maybe there is a way to fix it?]

**General information**
- Blender version: [e.g. 2.9.3]
- Addon version: [e.g. 1.3.0]
- Engine: [e.g. Evee or Cycles]
- Operating System: [e.g. Windows 10]

**Comment by karlisup** [comment]
Hi @HarmZero,
This is a known bug introduced by [Blender Core changes](https://developer.blender.org/T86719) in Blender 2.93 _alpha_. This is already prioritized, but as a workaround you can comment out contents of the function `set_current_values_as_defaults(self, context)` which can be located in your addons folder `C:\Users\{username}\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\physical-starlight-atmosphere\properties.py` line `15` and set it as follows:
```
def set_current_values_as_defaults(self, context):
    pass
    """Dynamically re-initialize atmosphere settings with new default values"""
    # print(self.preset_name)
    # updated_properties = update_atmosphere_settings()  # set new values for the properties
    # dattributes = {}
    # for property in updated_properties:
    #     propType = property["type"]
    #     propName = property["name"]
    #     if propType == "bool":
    #         dattributes[propName] = (bpy.props.BoolProperty, property["attributes"])
    #     elif propType == "int":
    #         dattributes[propName] = (bpy.props.IntProperty, property["attributes"])
    #     elif propType == "string":
    #         dattributes[propName] = (bpy.props.StringProperty, property["attributes"])
    #     elif propType == "float":
    #         dattributes[propName] = (bpy.props.FloatProperty, property["attributes"])
    #     elif propType == "floatvector":
    #         dattributes[propName] = (bpy.props.FloatVectorProperty, property["attributes"])
    #     elif propType == "enum":
    #         dattributes[propName] = (bpy.props.EnumProperty, property["attributes"])
    #
    # DynamicProperties = type('DynamicProperties', (PropertyGroup,), {'__annotations__': dattributes})
    # bpy.utils.register_class(DynamicProperties)
    # bpy.types.Scene.atmosphere_settings = PointerProperty(type=DynamicProperties)
```
This temporarily will disable setting new default values to preset default values, but it is not crucial functionality.

This should help, but the hotfix is on the way.

**Comment by karlisup** [comment]
Hi HarmZero,
Hot-fix has been released! It is fixed on PSA addon [v1.3.1](https://www.physicaladdons.com/psa/release-notes/#131-released-16042021). You can get your latest copy in the market you purchased it (either Blender market or Gumroad).
